ci-operator/config/openshift-kni/eco-ci-cd/.config.prowgen

```diff
@@ -13,3 +13,4 @@ slack_reporter:
   - telcov10n-sno-hub-ztp-f360
   - telcov10n-sno-spoke-f360
   - slcm-lab
+  - hcp-cnf
```

Comment:
place change it to the names of the jobs, I will comment on the names so you can find it easier.